[nrfconnect] Splitted building with Matter OTA and DFU over SMP (#56)
kkasperczyk-no/connectedhomeip: examples/lock-app/nrfconnect/README.md

#### Changing Device Firmware Upgrade configuration

> To change the default DFU configuration, edit some of the following overlay files depending on selected configuration:

Reviewer: ```suggestion
To change the default DFU configuration, edit the following overlay files corresponding to the selected configuration:
```
